.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: node_js
 
 node_js:
+  - 13
```

Comment:
Please add `node` to this list as well (or instead of 13) so that it's future-proofed.

Reply:
Done but when I get a chance I'm likely going to create a "tapjs/travis-ci" repository with shared travis configs similar to https://github.com/cfware/travis-ci (what I use for my repos).  This way testing config can be centrally managed for the whole tapjs org.